src/components/InjectedComponents/PostDummy.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react'
+import { usePostInfoDetails } from '../DataSource/usePostInfo'
+import { DefaultTypedMessageRenderer } from './TypedMessageRenderer'
+import { PluginUI } from '../../plugins/plugin'
+import { makeTypedMessageCompound, isTypedMessageSuspended, isTypedMessageKnown } from '../../protocols/typed-message'
+import { useValueRef } from '../../utils/hooks/useValueRef'
+import { currentWholePostVisibilitySettings, WholePostVisibility } from '../../settings/settings'
+
+export interface PostDummyProps {
+    zip?: () => void
+    unzip?: () => void
+}
+
+export function PostDummy(props: PostDummyProps) {
+    const parsedPostContent = usePostInfoDetails('parsedPostContent')
+    const postPayload = usePostInfoDetails('postPayload')
+    const wholePostVisibilitySettings = useValueRef(currentWholePostVisibilitySettings)
+
+    const processedPostMessage = Array.from(PluginUI.values()).reduce(
+        (x, plugin) => (plugin.postMessageProcessor ? plugin.postMessageProcessor(x) : x),
```

Comment:
`plugin.postMessageProcessor?.(x) ?? x` is much clear if the postMessageProcessor must not return undefined.